infra/src/main/resources/db/migration/prod/V93__fix_simplified_plan_polygons.sql

```diff
@@ -0,0 +1,12 @@
+alter table geometry.plan drop column bounding_polygon_simple;
+alter table geometry.plan add column bounding_polygon_simple postgis.geometry(polygon, 3067)
+  generated always as (
+    postgis.st_simplifypreservetopology(
```

Comment:
Jokseenkin rumempi tapa mutta tuntuisi tuottavan oikean tuloksen myös uudella versiolla. Vaikuttaisi että tuon st_buffer ihan dokumentaationsa mukaan pitäisikin tuottaa polygoniin leikattu sisäosa, mutta jostain syystä näin ei aiemmin käynyt. st_exterriorring ottaa sen polygonin joka me oikeasti halutaan mutta palauttaa sen linestringinä, joten siitä pitää vielä uudelleen tehdä polygon.